src/Components/fe-otp/test/fe-otp.test.js

```diff
@@ -8,13 +17,35 @@ describe('FeOtp', () => {
     element = await fixture(html`<fe-otp></fe-otp>`);
   });
 
-  it('the input validation event should be triggered on clicking submit', async () => {
+  it('Check if response is returned', async () => {
     const otpcode = element.shadowRoot.querySelector('#otp-code');
-    otpcode.modelValue = 'otp-code';
+    const notificationtag = element.shadowRoot.querySelector('fe-notification');
+    const requestMock = sinon.stub(ajax, 'requestJson');
+    requestMock.resolves({ body: 'SuccessResponse' });
 
+    otpcode.modelValue = '123456';
     setTimeout(() => element.triggerSubmit());
-    const { detail } = await oneEvent(element, 'input-validation');
-    expect(detail, { otpcode: 'otpcode' }).to.exist;
+    const { detail } = await oneEvent(element, 'complete');
+    assert.deepEqual(detail, {
```

Comment:
update the testcase based on previous coment

Reply:
Okay I will update